study.md

```diff
@@ -33,7 +33,7 @@ How does Ember use the URL to load view-state? Which layers in Ember are
 responsible for which tasks?
 
 ```md
-<!-- your answer here -->
+The Ember URL is matched by the route and the route connects to the appropriate model. Afterward the controller and template uses the model to render the the contents on the page. In addition, the router manages the URL with history, hash, auto, and none options.
```

Comment:
One modification/clarification. The router is the layer that gets a URL string and uses it to load a particular page. The `location` option simply tells it how it should parse the URL.